src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -242,8 +242,8 @@ export class DebuggerVariables extends DebugLocationTracker
         } else if (
             message.type === 'response' &&
             message.command === 'variables' &&
-            message.body &&
-            this.currentSeqNumsForVariables.has(message.request_seq)
+            message.body
+            // this.currentSeqNumsForVariables.has(message.request_seq)
```

Comment:
This messes up variable hover. Vandy added this to fix variable hover.
https://github.com/microsoft/vscode-jupyter/pull/6186/files

Reply:
done